src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -102,6 +106,22 @@ public CommandResult executeUndoableCommand() throws CommandException {
     }
 
     /**
+     * Does the appropriate list status checks on {@code personToEdit}
+     * and changes the list status of {@code editedPerson} accordingly.
+     * Returns {@code editedPerson}.
+     */
+    private Person listSyncChecks(ReadOnlyPerson personToEdit, Person editedPerson) {
+        if (personToEdit.getIsWhitelisted() && editedPerson.getDebt().toNumber() > 0) {
+            editedPerson.setIsWhitelisted(false);
+        } else if (!personToEdit.getIsWhitelisted() && !personToEdit.getIsBlacklisted()
```

Comment:
I feel that `getIsWhitelisted()` and `getIsBlacklisted()` can be renamed for clarity. Assuming that you are checking if `presonToEdit` is whitelisted or blacklisted, the method names could be `IsWhitelisted()` and `IsBlacklisted()`

Reply:
agreed, it is okay for a method and variable to have a same name by the way, so it is okay to call them as jel suggested

Reply:
Yeap thanks guys. Noted.
Jel you mean `isWhitelisted()` or `IsWhitelisted()`? It's the former right?

Reply:
yes